### ISSUE TITLE
Unable to see brown keyword in diff editor using Kimbie Dark theme

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#282526]
1. Use Kimbie Dark.
2. Wrap some code in a try-catch.
3. View the diff.
4. Place cursor on the `try` line.
5. :bug: The green clashes with the try keyword. I think the green needs to change.

<img width="270" height="84" alt="Image" src="https://github.com/user-attachments/assets/d3b89d88-d147-4f1d-a491-1ad1ec24363a" />
